project/JPMSPlugin.scala

```diff
@@ -114,8 +114,6 @@ object JPMSPlugin extends AutoPlugin {
         )
     }.toSeq
 
-    ensureDirectoriesExist(modulePath, log)
```

Comment:
These warnings are annoying and they are not useful. Java does not mandate that the directories passed into, e.g., `--module-path` option exist.